google-cloud-speech/lib/google/cloud/speech/v1/speech_client_config.json

```diff
@@ -1,40 +1,40 @@
 {
   "interfaces": {
-    "google.cloud.speech.v1beta1.Speech": {
+    "google.cloud.speech.v1.Speech": {
       "retry_codes": {
-        "idempotent": [
-          "DEADLINE_EXCEEDED",
-          "UNAVAILABLE"
-        ],
-        "non_idempotent": [
-          "UNAVAILABLE"
-        ]
+          "idempotent": [
```

Comment:
All these files are all generated by an internal Google tool called GAPIC. The values in this specific file come from the file `google/cloud/speech/v1/cloud_speech_gapic.yaml` in the [googleapis/googleapis](https://github.com/googleapis/googleapis) repo ([link](https://github.com/googleapis/googleapis/blob/master/google/cloud/speech/v1/cloud_speech_gapic.yaml).

I can't point you to documentation on how GAPIC works or why these values were used. Perhaps @swcloud can help here?